docs/users/faq/NexusModsDownloads.md

```diff
@@ -13,6 +13,23 @@ On a Linux PC, you can check the default app for `nxm://` downloads and reset it
 - Restart any open web browser windows to ensure the change is applied.
 - In your browser, when selecting what application should be used to open the nxm link, select **NexusMods.App NXM Handler**.
 
+### Setting up Gear Lever on Linux
+For Linux users who have the AppImage version of the app and are unable to resolve the registration issue using the commands above, we recommend [Gear Lever](https://github.com/mijorus/gearlever).
+
+1. Download and install Gear Level, preferable via a package manager such as [Flathub](https://flathub.org/en/apps/it.mijorus.gearlever). 
```

Comment:
This should say 'Gear Lever', not 'Gear Level'

Reply:
Fixed in upcoming commit, thanks.